env/GAEAC6.env

```diff
@@ -227,7 +227,15 @@ case ${step} in
  ;;
  "fcst" | "efcs")
 
-    export OMP_STACKSIZE=512M
+    export OMP_NUM_THREADS=1
```

Comment:
`OMP_NUM_THREADS` should not be set here.  It is correctly being set/unset [here](https://github.com/NOAA-EMC/global-workflow/blob/a32690bd654d3bca4e394021c9d3951945543c18/scripts/exglobal_forecast.sh#L174-L178)